src/index.test.js

```diff
@@ -79,6 +73,81 @@ describe('index', () => {
     expect(actualTags).toEqual([{ key: 'k0', value: 'v0' }, { key: 'k1', value: 'v1' }]);
   });
 
+  test('logs - error (should filter long entries and cut after the 10s element)', async () => {
+    const consoleLog = jest.spyOn(console, 'log');
+    const lumigoImport = require('./index');
+    const lumigo = lumigoImport({ token: 'T' });
+    const name = '1'.repeat(1000);
+    let msg =
+      '{"message":"This is error message","type":"ClientError","level":40,"extra":{"a":"3","b":"true","c":"aaa","d":"[object Object]","aa":"a","a0":"a0","a1":"a1","a2":"a2","a3":"a3","a4":"a4"}}';
+    const maxelementsinextra = MAX_ELEMENTS_IN_EXTRA;
+    let extra = {
+      a: 3,
+      b: true,
+      c: 'aaa',
+      d: {},
+      aa: 'a',
+      [name]: name,
+      ...Object.fromEntries([...Array(maxelementsinextra).keys()].map(k => [`a${k}`, `a${k}`])),
+    };
+    lumigo.error('This is error message', {
+      type: 'ClientError',
+      extra,
+      err: new TypeError('This is type error'),
+    });
+    expect(consoleLog).toBeCalledWith(`[LUMIGO_LOG] ${msg}`);
+  });
+
+  test('err with type and exception', async () => {
+    const consoleLog = jest.spyOn(console, 'log');
+    const lumigoImport = require('./index');
+    const lumigo = lumigoImport({ token: 'T' });
```

Comment:
can you try and reduce the code duplication?
These 3 lines repeat in all the tests